api/proto/teleport/okta/v1/okta_plugin_config_helper_service.proto

```diff
@@ -0,0 +1,79 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.okta.v1;
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/okta/v1;oktav1";
+
+// OktaPluginConfigHelperService provides methods that will help with plugin configuration.
+// This service can only be contacted by the proxy.
+service OktaPluginConfigHelperService {
```

Comment:
Do we need the whole gRPC service for that ? 

For my understending we need to expose REST endpoint for FE to fetch okta apps groups using okta token/org_id and gRPC service seems to be superfluous for that. 

 What would you say about creating just internal service that will be used in scope of Teleport Proxy ?

Reply:
I think you're right, but I got concerned about being able to actually verify that the was the proxy user. I'll can this and make it proxy only.